Godeps/_workspace/src/k8s.io/kubernetes/pkg/apiserver/handlers.go

```diff
@@ -439,19 +443,32 @@ type APIRequestInfoResolver struct {
 // /api/{version}/redirect/{resource}/{resourceName}
 // /api/{version}/watch/{resource}
 // /api/{version}/watch/namespaces/{namespace}/{resource}
-func (r *APIRequestInfoResolver) GetAPIRequestInfo(req *http.Request) (APIRequestInfo, error) {
-	requestInfo := APIRequestInfo{
-		Raw:  splitPath(req.URL.Path),
-		Verb: strings.ToLower(req.Method),
+//
+// NonResource paths
+// /apis/{api-group}/{version}
+// /apis/{api-group}
+// /apis
+// /api/{version}
+// /api
+// /healthz
+// /
```

Comment:
This comment looks prone to staleness since the list of non resource paths is driven by injection/policy.